components/testmonitor/sources/common/base.mjs

```diff
@@ -0,0 +1,83 @@
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+import moment from "moment";
```

Comment:
The package.json file should include `moment` as a dependency. You can add it by running `npm install moment` in the /components/testmonitor directory.